tests/libmigration/migration.go

```diff
@@ -37,6 +37,22 @@ import (
 
 const MigrationWaitTime = 240
 
+func NewMigration(vmiName string, namespace string) *v1.VirtualMachineInstanceMigration {
```

Comment:
What about calling it just `New`? The `libmigration` already explains what it is about. (similar to `libvmi.New()`)

Reply:
I like that! Done! Thanks